python_modules/dagster-test/dagster_test/test_project/build.sh

```diff
@@ -36,16 +36,19 @@ mkdir -p modules
 cp $GOOGLE_APPLICATION_CREDENTIALS ./modules/gac.json
 
 echo -e "--- \033[32m:truck: Copying files...\033[0m"
-alias copy_py="rsync -av \
-      --progress \
-      --exclude *.egginfo \
-      --exclude *.tox \
-      --exclude dist \
-      --exclude __pycache__ \
-      --exclude *.pyc \
-      --exclude .coverage"
-
-copy_py $ROOT/python_modules/dagster \
+
+echo "ROOT IS $ROOT"
+
+rsync \
```

Comment:
aliasing wasn't working for me locally, we only use alias once so just drop in rsync here